lib/makara/middleware.rb

```diff
@@ -89,6 +89,21 @@ def makara_values(env)
       [nil, nil]
     end
 
+    # determine how lone the cookie TTL should last
+    # this needs to be done huristically, as we cannot be sure which connection 
+    # would have had the longest timeout from this middleware
+    def cookie_ttl
```

Comment:
As far as I can tell, we can't use the Makara::ConfigParser here.   There may be many instances of Makara in play (ActiveRecord vs ActiveRabbit, so we won't know which connection to source).  I also don't know how we would find the instance at the middleware level.  

That's why I tried to hack it with ActiveRecord::Base...